includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1395,6 +1395,11 @@ public function cancel_authorization( $order ) {
 	 * @return array          The level 3 data to send to Stripe.
 	 */
 	public function get_level3_data_from_order( $order ) {
+		// Non-US merchants do not need to send level3 data.
+		if ( 'US' !== $this->account->get_account_country() ) {
```

Comment:
I raised a question in wcpay channel as not 100% sure if we need to verify the country from order as well.

Reply:
Confirmed, both merchant and customer countries need to be in the US.

Reply:
There is already [a check at the bottom of `get_level3_data_from_order()`](https://github.com/Automattic/woocommerce-payments/pull/1400/files#diff-bcfd303d507c33a27671935e724ee64d0df30ff2a18bb6ef4c94f28d48190124R1445-R1449) to make sure we only pass customer's zip code if it's a US zip code.

Reply:
:+1: